configs/developers-cloudflare.json

```diff
@@ -57,6 +58,7 @@
     "https://developers.cloudflare.com/network-interconnect/sitemap.xml",
     "https://developers.cloudflare.com/pages/sitemap.xml",
     "https://developers.cloudflare.com/page-shield/sitemap.xml",
+    "https://developers.cloudflare.com/railgun/",
```

Comment:
@marciocloudflare You're missing the filename in the URL (`sitemap.xml`).

Reply:
I'm on it. Thank you!

Reply:
Good catch!